libs/client-sdk/src/client_impl.hpp

```diff
@@ -1,33 +1,32 @@
 #pragma once
 
```

Comment:
Sorting include paths according to our standard from most specific to least specific.